core/systems/assignment/content_page_form.ex

```diff
@@ -99,16 +111,34 @@ defmodule Systems.Assignment.ContentPageForm do
   end
 
   @impl true
-  def handle_event("update", %{status: :off}, %{assigns: %{page_ref: page_ref}} = socket) do
-    {:ok, _} = Assignment.Public.delete_page_ref(page_ref)
-
+  def handle_event("update", %{status: :off}, socket) do
     {
       :noreply,
-      socket |> assign(page_ref: page_ref)
```

Comment:
Consider to add an extra check on wysiwyg field text. If no text, there is no real reason for the confirmation.